midas2root/midas2root_ptf.cxx

```diff
@@ -14,20 +14,22 @@
 #include <TTree.h>
 #include <TFile.h>
 
-#define TRUE 1
-#define FALSE 0
 
-#define nPoints_max 6000 // John 2019-11-05 Reduced from 1000000
-#define num_phidg_max 10000
-#define max_temp_sensor 20
-#define num_v1730_max 70 // IMPOTANT: IF THIS IS EVER CHANGED, ALSO CHANGE THE HARDCODED VALUES FOR WAVEFORM BRANCH WIDTHS AS WELL (see: "v1730 data")
-#define timeStart 130 // defines start of PMT Pulse timing window, currently at the 130th sample of 200, with a window size of 70 samples.
+constexpr bool single_point = false; // set to true for non-scans. This makes it accept everything! 
+constexpr int nPoints_max = single_point ? 1800*1600 : 12000; // if single_point, we do an hour. otherwise, like 4 seconds of data
+
+const int num_phidg_max = 10000;
+const int max_temp_sensor = 20;
+const int num_v1730_max = 140; // IMPOTANT: IF THIS IS EVER CHANGED, ALSO CHANGE THE HARDCODED VALUES FOR WAVEFORM BRANCH WIDTHS AS WELL (see: "v1730 data")
```

Comment:
This is broadly fine, but obviously does make the file sizes twice as big.  Is this really needed?

Reply:
I suppose not - this was done to make sure a time shift wasn't causing lost pulses in some cases. I can revert this to a 70-sample wide waveform after double checking the timing.